File: Simulator-CSharp/Models/Card.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ATM.Models
{
    public class Card
    {

        public string Number { get; set; }
        public string Password { get; set; }

        public string Bank { get; set; }

        public System.DateTime Exp { get; set; }

        public bool Locked { get; set; }

        public bool IsExpired()
        {
            return Exp.Date.Subtract(DateTime.Now.Date).Days <= 0;
        }

        public string Account1 { get; set; }
        public string Account2 { get; set; }

    }
}

File: Simulator-VB/Models/Card.vb
Namespace Models

    Public Class Card

        Public Property Number As String
        Public Property Password As String

        Public Property Hash As String

        Public Property Bank As String

        Public Property Exp As Date

        Public Property Locked As Boolean

        Public Function IsExpired() As Boolean
            Return Exp.Date.Subtract(Now.Date).Days <= 0
        End Function

        Public Property Account1 As String
        Public Property Account2 As String

    End Class

End Namespace